sql/telemetry/origin_content_blocking/view.sql

```diff
@@ -0,0 +1,28 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod.telemetry.origin_content_blocking` AS
+SELECT
+  submission_date,
+  batch_id,
+  origin,
+  aggregate
+FROM
+  (
+    SELECT
+      *,
+      ROW_NUMBER() OVER (
+        PARTITION BY
+          submission_date,
+          batch_id,
+          origin
+        ORDER BY
+          TIMESTAMP DESC
+      ) AS insert_rank
+    FROM
+      `moz-fx-data-shared-prod.telemetry_derived.origin_content_blocking`
+  )
+WHERE
+  insert_rank = 1
```

Comment:
this is inconsistent with this pattern's use elsewhere in the repo. I think we should:
1. rename `insert_rank` as `_n`
2. make the select outer select clause `SELECT * EXCEPT(_n)`
3. move the inner query to a CTE